### ISSUE TITLE
Is this repo up to date? 

### ISSUE THREAD

**Issue by spinny7z** [codemzy/mern-auth#7]
Is this all up to date in 2021?

**Comment by codemzy** [comment]
No, it's never been released and was last updated 3 years ago. While I did plan to (and still may) update and release it at some stage, I'm not currently working on it.
